File: SDKs/amazonaws.com/codedeploy/2014-10-06/typescript/src/sdk/models/shared/elbinfo.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ElbInfo
/** 
 * Information about a load balancer in Elastic Load Balancing to use in a deployment. Instances are registered directly with a load balancer, and traffic is routed to the load balancer.
**/
export class ElbInfo extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=name" })
  name?: string;
}

File: SDKs/amazonaws.com/codedeploy/2014-10-06/python/src/sdk/models/shared/elbinfo.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class ElbInfo:
    r"""ElbInfo
    Information about a load balancer in Elastic Load Balancing to use in a deployment. Instances are registered directly with a load balancer, and traffic is routed to the load balancer.
    """
    
    name: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('name') }})
    
